pom.xml

```diff
@@ -30,31 +30,29 @@
     <parent>
         <groupId>io.gravitee</groupId>
         <artifactId>gravitee-parent</artifactId>
-        <version>20.5</version>
+        <version>22.2.4</version>
     </parent>
 
     <properties>
-        <gravitee-bom.version>3.0.27</gravitee-bom.version>
-        <gravitee-apim.version>3.20.32</gravitee-apim.version>
-        <gravitee-gateway-api.version>2.0.1</gravitee-gateway-api.version>
-        <gravitee-policy-api.version>1.11.0</gravitee-policy-api.version>
+        <gravitee-apim.version>4.6.0</gravitee-apim.version>
+        <gravitee-reactor-message.version>5.0.0</gravitee-reactor-message.version>
         <gravitee-common.version>2.3.0</gravitee-common.version>
```

Comment:
you can remove this as it comes from gravitee-apim-bom Can you remove all the properties that are not use